src/numerics/petsc_matrix.C

```diff
@@ -1002,7 +1010,18 @@ numeric_index_type PetscMatrix<T>::m () const
   return static_cast<numeric_index_type>(petsc_m);
 }
 
+template <typename T>
+numeric_index_type PetscMatrix<T>::local_m () const
+{
+  libmesh_assert (this->initialized());
+
+  PetscInt m = 0;
 
+  auto ierr = MatGetLocalSize (_mat, &m, PETSC_NULL);
```

Comment:
Minor issue: we just simply use NULL in PETSc right now. We were using PETSC_NULL, but not any more.

Reply:
Haha, really? I guess we should actually be changing them all to `nullptr` now but I don't really care either way. Was `PETSC_NULL` ever not `NULL`? What issue was it used to address?

Reply:
I have no idea whey we were using PETSC_NULL. I still have a lot of PETSC_NULL in my PhD code:-)

In PETSc, we have this:

`#define PETSC_NULL NULL`

But most of PETSc code is using `NULL`

Cody forced me to use `nullptr` for PETSc code in MOOSE.